### DISCUSSION TITLE
Typos

### DISCUSSION THREAD

**Discussion by Lairo** [headfirstpython/third#2] [General]
Hey there, I am going through the book ty, btw, for taking the time to writing it. I have noticed some typos if you don't mind, I can add some later on as I read through.

I have Third Edition 2023-08-02 copy

Page xxxi
before "We'll" there is a random letter 't' at the start of the sentence

Page 3 4th python answer
typo - I'm Ta'L'king  the L is missing
 
 Page 52
 down at the bottom 
 It's often useful to 'T'hink ....  letter t is missing. there's also extra white space in the same sentence "hink of__the". (that one could just be print..)

**Comment by headfirstpython** [comment]
Thanks for those, but please also see:
https://www.oreilly.com/catalog/errata.csp?isbn=0636920252207&order=date for the full
list (so far).  👍

On Mon, 23 Oct 2023 at 17:39, Lairo ***@***.***> wrote:

> Hey there, I am going through the book ty, btw, for taking the time to
> writing it. I have noticed some typos if you don't mind, I can add some
> later on as I read through.
>
> I have Third Edition 2023-08-02 copy
>
> Page xxxi
> before "We'll" there is a random letter 't' at the start of the sentence
>
> Page 3 4th python answer
> typo - I'm Ta'L'king the L is missing
>
> Page 52
> down at the bottom
> It's often useful to 'T'hink .... letter t is missing. there's also extra
> white space in the same sentence "hink of__the". (that one could just be
> print..)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/headfirstpython/third/discussions/2>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7O3GTGIT2NEZWQGBW2JG7TYA2MTDAVCNFSM6AAAAAA6MLUZOKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVG43DQOJUGQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Paul Barry

BSc (Hons), MSc, PG Cert (T&L)

Lecturer - Department of Computing
E ***@***.*** | *setu.ie* <https://setu.ie> | work
E ***@***.*** | personal
[image: turn on images] <https://setu.ie>

Kilkenny Road Campus, Kilkenny Road, Carlow, R93 V960, Ireland
Campas Bhóthar Chill Chainnigh, Bóthar Chill Chainnigh, Ceatharlach, R93
V960, Éire

**Comment by Lairo** [comment]
ah I see, do you want me to continue or is there no need sts?

**Comment by headfirstpython** [comment]
If you find any more typos (or errors), check the O'Reilly list to make
sure it has not been previously reported.  If it has not, please add to the
errata so that I can confirm it.  All of your submissions and comments are
greatly appreciated.  Thank you.  😊

On Mon, 23 Oct 2023 at 19:10, Lairo ***@***.***> wrote:

> ah I see, do you want me to continue or is there no need sts?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/headfirstpython/third/discussions/2#discussioncomment-7361976>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7O3GTHJP2RIMCCEL22NLYDYA2XKHAVCNFSM6AAAAAA6MLUZOKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGNRRHE3TM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Paul Barry

BSc (Hons), MSc, PG Cert (T&L)

Lecturer - Department of Computing
E ***@***.*** | *setu.ie* <https://setu.ie> | work
E ***@***.*** | personal
[image: turn on images] <https://setu.ie>

Kilkenny Road Campus, Kilkenny Road, Carlow, R93 V960, Ireland
Campas Bhóthar Chill Chainnigh, Bóthar Chill Chainnigh, Ceatharlach, R93
V960, Éire
